packages/core/src/interaction-output.ts

```diff
@@ -32,20 +32,23 @@ export class InteractionOutput implements WoT.InteractionOutput {
     private content: Content;
     private parsedValue: unknown;
     private buffer?: ArrayBuffer;
-    data?: ReadableStream;
+    private _stream?: ReadableStream;
     dataUsed: boolean;
     form?: WoT.Form;
     schema?: WoT.DataSchema;
 
+    public get data(): ReadableStream {
+        if (this._stream) {
```

Comment:
`this.dataUsed = true; ` // don't we need to set that ?

Reply:
Technically, it should be set only after somebody reads the stream... but given the behavior above, we could do that!